diary.go

```diff
@@ -7,13 +7,16 @@ import (
 	"io"
 	"os"
 	"time"
+
+	"github.com/go-stack/stack"
```

Comment:
Any reason not to do this inline rather than pulling in a dependency?    I like having no 3rd party dependencies.  We'd just need to pull in this function, right? https://github.com/go-stack/stack/blob/master/stack.go#L31